.github/workflows/ci.yml

```diff
@@ -80,9 +80,7 @@ jobs:
         # See https://github.com/pypa/pip/issues/11982 for details
         python -m pip install --upgrade "pip!=23.1"
         python -m pip install --upgrade setuptools
-        # We don't actually want to install briefcase; we just
-        # want the dev extras so we have a known version of tox.
-        python -m pip install $(ls dist/briefcase-*.whl)[dev]
+        python tests/env_utils.py install-req tox
```

Comment:
This will likely be opaque enough for first-time readers (or, lets be honest, me in 2 months :-) that it's worth retaining a comment so it's clear what this is doing.